LoopKitTests/VersionUpdateTests.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  VersionUpdateTests.swift
```

Comment:
[nit] Might want to name this file `VersionCheckServiceTests.swift`. When looking for associated tests to a particular file I look for the same file with the `Tests` suffix. In this case I wouldn't know these tests existed .